### PR TITLE
combine aliases page with members page

### PR DESCRIPTION
**Pull request by staltz**
Tests are missing (I'm working on them now), but wanted to submit this already for review of the implementation, and to let @cblgh see the HTML dropdown using `<details>`.

### Members page

(notice no Aliases option on the left-side menu)

Screenshot from 2021-04-07 14-21-36

### Member details page

Screenshot from 2021-04-07 14-21-45

## Mobile

Looks good on mobile too, but didn't feel it was necessary to screenshot here.

### CODE REVIEW

File: web/templates/admin/member.tmpl [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  <h1
+    class="text-3xl tracking-tight font-black text-black mt-2 mb-4"
+  >{{i18n "AdminMemberDetailsTitle"}}</h1>
+
+  <label class="mt-2 mb-1 font-bold text-gray-400 text-sm">{{i18n "AdminMemberDetailsSSBID"}}</label>
+  <p class="mb-8 font-mono font-bold tracking-wider truncate text-gray-900">{{.Member.PubKey.Ref}}</p>
+
+  <label class="mt-2 mb-1 font-bold text-gray-400 text-sm">{{i18n "AdminMemberDetailsRole"}}</label>
+  <details class="mb-8 self-start w-40">
+    <summary class="px-3 py-1 rounded shadow bg-white ring-1 ring-gray-300 hover:bg-gray-100 cursor-pointer">
```
Comment (staltz) [inline web/templates/admin/member.tmpl:12]: @cblgh yolo

File: roomdb/sqlite/members.go [hunk 2]
```diff
@@ -82,16 +97,18 @@ func (m Members) GetByFeed(ctx context.Context, h refs.FeedRef) (roomdb.Member,
...
-	for i, listEntry := range all {
-		members[i].ID = listEntry.ID
-		members[i].Role = roomdb.Role(listEntry.Role)
-		members[i].PubKey = listEntry.PubKey.FeedRef
+	for i, entry := range all {
+		members[i].ID = entry.ID
+		members[i].Role = roomdb.Role(entry.Role)
+		members[i].PubKey = entry.PubKey.FeedRef
+		members[i].Aliases = getAliases(entry)
+		println(len(members[i].Aliases))
```
Comment (cryptix) [inline roomdb/sqlite/members.go:111]:
```suggestion
```

File: web/handlers/admin/members.go [hunk 3]
```diff
@@ -104,7 +104,8 @@ func (h membersHandler) changeRole(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	http.Redirect(w, req, redirectToMembers, http.StatusTemporaryRedirect)
+	memberDetailsURL := fmt.Sprint("/admin/member?id=", memberID)
```
Comment (cryptix) [inline web/handlers/admin/members.go:107]:
```suggestion
	urlTo := web.NewURLTo(router.CompleteApp())
	memberDetailsURL := urlTo(router.AdminMemberDetails, "id", memberID).String()
```

this way we are safe when we change the layout of the paths.

Comment (cryptix) [reply web/handlers/admin/members.go:107]: might need to add imports for package `.../web` and `.../web/router` but can't suggest changes outside of the diff... :upside_down_face:

**Review by cryptix — COMMENTED** [review]
just two nitpicks about the URL building and a dangling `println` but otherwise this looks sweet!

File: web/handlers/admin/members.go (hunk 3 above)
Comment (staltz) [reply web/handlers/admin/members.go]: Thanks for the suggestion. I did it in a separate commit, and added the imports

**Comment by staltz** [comment]
Can haz merge? (My next PR is building on top of this branch)

**Comment by cryptix** [comment]
purrfect! go ahead!

**Pull request merged**